proxy/src/http/source.rs

```diff
@@ -349,10 +354,12 @@ mod handler {
     pub async fn branches(
         peer: Arc<Mutex<coco::PeerApi>>,
         project_urn: coco::Urn,
+        super::PeerQuery { peer_id }: super::PeerQuery,
     ) -> Result<impl Reply, Rejection> {
+        log::info!("peer.id = {:?}", peer_id);
```

Comment:
```suggestion
```

Reply:
I mean... I'm sort of interested in logging in the endpoints. But maybe I'll create a ticket for it.